projects/openapi-utilities/src/openapi3/traverser.ts

```diff
@@ -0,0 +1,505 @@
+import { jsonPointerHelpers as jsonPointer } from '@useoptic/json-pointer-helpers';
```

Comment:
This is straight copy pasted from the other traverser, but only emits JSON paths + where it comes from.

We can extra enough data from here from the JSON path + type to get:
- location
- actual values

Reply:
Also, see comment here https://github.com/opticdev/optic/pull/1495#discussion_r1068574852 - had to add a `path` fact type due to how the diff works and how we need to interpret the diff